frontend/public/components/overview/pods-overview.tsx

```diff
@@ -251,10 +251,12 @@ export const PodsOverviewMultiple: React.SFC<PodsOverviewMultipleProps> = ({
       Object.keys(resources).length > 0 &&
       Object.keys(resources).every((key) => resources[key].loaded)
     ) {
-      let updatedPods = podResources.reduce((acc, resource) => {
-        acc.push(...getPodsForResource(resource, resources));
-        return acc;
-      }, []);
+      let updatedPods = podResources?.length
+        ? podResources.reduce((acc, resource) => {
+            acc.push(...getPodsForResource(resource, resources));
+            return acc;
+          }, [])
+        : [];
```

Comment:
I personally would go with this, but its okay :smirk: 

```suggestion
      let updatedPods = podResources?.reduce((acc, resource) => {
        acc.push(...getPodsForResource(resource, resources));
        return acc;
      }, []) ?? [];
```